src/MSBuild/XMake.cs

```diff
@@ -3539,7 +3539,7 @@ private static void ShowHelpPrompt()
         /// </summary>
         private static void ShowVersion()
         {
-            Console.Write(ProjectCollection.Version.ToString());
+            Console.WriteLine(ThisAssembly.AssemblyInformationalVersion);
```

Comment:
This one I'm more worried about. Think people are parsing the output of `msbuild /nologo /version` and will be broken by a non-`System.Version` compatible output? I suspect they might :(

Reply:
agreed